docs/writing-schemas/nested-arguments.md

```diff
@@ -0,0 +1,100 @@
+---
+id: nested-arguments
+title: Nested Arguments
```

Comment:
i think this should be renamed to nested fields?

Reply:
For ease of maintenance going forward, it's easiest if the id and file name both match the title. Makes it easier to find the right file when users report a bug based on the page title, which is the only part they can see. You'll have to update links to this page if you rename it. You'll have to update website/sidebars.json if you change the id.

In fact, you need to add this to website/sidebars.json in any case, it's not there yet.